data/thebarshablog/events/2023-02-21-Decoding-CFPs-Twitter-Space.json

```diff
@@ -0,0 +1,12 @@
+{
+  "isVirtual": true,
+  "isInPerson": false,
+  "color": "purple",
+  "name": "Decoding CFPs (Call for Proposals)",
+  "description": "Twitter Space",
+  "date": {
+    "start": "2023-02-21 08:00:00 IST",
+    "end": "2023-02-21 09:30:00 IST"
```

Comment:
I don't think that is a valid date/time

Please see the documentation for examples `2022-12-09T16:00:00.000+00:00`
http://localhost:3000/docs/how-to-guides/events